src/SearchField/index.scss

```diff
@@ -8,7 +8,10 @@
     display: flex;
     align-items: center;
     width: 100%;
-    background-color: $search-form-background-color;
+  }
+
+  .btn {
```

Comment:
Can we create a new class for this button so that we don't have conflicts with other buttons in the component?